providers/implementations/ciphers/build.info

```diff
@@ -63,8 +63,12 @@ IF[{- !$disabled{des} -}]
       cipher_tdes_default.c cipher_tdes_default_hw.c \
       cipher_tdes_wrap.c cipher_tdes_wrap_hw.c
   SOURCE[$DES_GOAL]=\
-      cipher_desx.c cipher_desx_hw.c cipher_tdes_common.c\
+      cipher_desx.c cipher_desx_hw.c \
       cipher_des.c cipher_des_hw.c
+ IF[{- !$disabled{shared} -}]
```

Comment:
Please use the same condition as is used in `providers/build.info` to determine if the legacy provider is going to be a module or part of libcrypto.

Reply:
OK, done.